app/helpers/carts_helper.rb

```diff
@@ -0,0 +1,34 @@
+module CartsHelper
+  def current_carts
+    session[:carts] ||= {}
+  end
+
+  def price_item_cart price, quantity
+    quantity * price
+  end
+
+  def total_money_cart
+    total = 0
+    current_carts.each do |id, quantity|
+      product = load_product_in_cart id
+      total_item = price_item_cart product.price, quantity.to_i
+      total += total_item
+    end
+    total
+  end
+
+  def load_product_in_cart id
+    product = Product.find_by(id: id)
+    unless product
+      flash[:danger] = t "not_found"
+      delete_item_cart id
+    end
+    product
```

Comment:
```suggestion
    return product if product.present?

    flash[:danger] = t "not_found"
      delete_item_cart id

```
Sử dụng guar clause nhé